### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for "Volume control slider" in the YouTube player

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#33]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/docs/introvideos/overview ](https://code.visualstudio.com/docs/introvideos/overview )in EDGE
2. Navigate to any video listed under "Introductory Videos"
3. Select any video and press Enter key
4. Play the video in the redirected page
5. Naviagte to the volume control slider option in the video.
6. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Slider"

**Expected Result:**
"Name" property should indicate the purpose of control. So, Name property should be meaningful 
Eg: "Volume Control" slider

**User Impact:**
Visually impaired users  will not get to know the purpose of the link as the screen reader will not announce anything when the user arrives at the control. Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshots:**
![name property is empty for volume control slider](https://cloud.githubusercontent.com/assets/24825441/21709812/028539f0-d40a-11e6-88fd-f25d111283b1.png)

**Comment by chrisdias** [comment]
On my windows box it says "slider at XX percent volume" when using narrator. As a result, this is "no repro".
